src/styles/CopyOverlay.js

```diff
@@ -35,7 +35,7 @@ export default {
 		fontSize       : '4rem',
 		color          : '#fff',
 		transform      : 'scale(0.1)',
-		opacity        : '0',
+		visibility     : 'hidden',
```

Comment:
```suggestion
		zIndex : '-1',
```
Do you think it is the best way to solve it? 
May be you try to use z-index?

Reply:
Hidden will assure that wont be visible, z-index on the other hand depend on elements zIndex to be right. Unless you want a opacity transition, then z-index should be the solution.